Documentation/contributing.rst

```diff
@@ -1661,6 +1661,9 @@ This process applies to all releases other than minor releases, this includes:
 If you intent to release a new minor release, see the
 :ref:`minor_release_process` section instead.
 
+.. note:: The following commands have been validated when ran in the VM
+          used in the Cilium development process. See :ref:`dev_env` for
+          detailed instructions about setting up said VM.
```

Comment:
Why is this important to note? Specifically because it's run inside the dev VM rather than on OSX? (It's fine, I just thought "the commands *should* have been validated before we put them into the documentation)

Reply:
I like to specify that we have noted that these steps work in a specified environment that we provide, like we do for any part of the development process. The backport checker scripts do not work on my Mac based off of prior experience.

Reply:
Some of our make targets also won't work on osx. Maybe we should stick the "works only with linux" comment at the top of the contributor guide?